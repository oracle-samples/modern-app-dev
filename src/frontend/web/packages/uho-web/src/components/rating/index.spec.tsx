/*
 * Copyright (c) 2023 Oracle and/or its affiliates.
 * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracl.com/licenses/upl/
 */
import { Rating } from './index';
import { h } from 'preact';
import { snapshot } from '../../../test/jetHelper';

snapshot('Rating', <Rating negative={{ count: 10, text: 'Negative' }} positive={{ count: 5, text: 'Positive' }} />);
