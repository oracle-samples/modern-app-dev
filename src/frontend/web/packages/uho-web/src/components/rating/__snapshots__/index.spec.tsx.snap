// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests for Rating Snapshot: Component - Rating 1`] = `
<DocumentFragment>
  <div
    data-oj-binding-provider="preact"
  >
    <oj-picto-chart
      class="oj-dvtbase oj-pictochart oj-component-initnode oj-complete"
    >
      <div
        class="oj-helper-detect-contraction"
      >
        <div
          style="width: 200%; height: 200%;"
        />
      </div>
      <div
        class="oj-helper-detect-expansion"
      >
        <div />
      </div>
      <template
        slot="itemTemplate"
      />
      <div
        style="visibility: hidden;"
      />
      <svg
        height="100%"
        width="100%"
      >
        <defs />
        <g
          cursor="default"
          font-family="-apple-system,BlinkMacSystemFont,'Segoe UI','Helvetica Neue',Arial,sans-serif"
          font-size="12px"
          font-weight="400"
        >
          <g />
        </g>
      </svg>
    </oj-picto-chart>
  </div>
  <div
    id="__oj_layerhost_container"
  />
</DocumentFragment>
`;
